data/mysql/schema.sql

```diff
@@ -764,7 +766,8 @@ CREATE TABLE `ezcontentobject_tree` (
   KEY `ezcontentobject_tree_p_node_id` (`parent_node_id`),
   KEY `ezcontentobject_tree_path` (`path_string`),
   KEY `ezcontentobject_tree_path_ident` (`path_identification_string`(50)),
-  KEY `modified_subnode` (`modified_subnode`)
+  KEY `modified_subnode` (`modified_subnode`),
+  KEY `ezcontentobject_tree_contentobject_id_path_string` (`contentobject_id`, `path_string`)
```

Comment:
same: you might want to put path_string 1st

Reply:
> you might want to put path_string 1st

Indeed, on live data that might make a difference. Thanks, fixed.